.travis.yml

```diff
@@ -61,6 +61,10 @@ before_install:
   - if [[ $TRAVIS_OS_NAME == 'osx' ]]; then brew update; brew install python; brew link --overwrite python ; fi
   - pip install -U --user pip
   - pip install -U --user virtualenv
+  # Fail fast on lint errors
+  - pip install -U --user flake8
+  - if [[ $TRAVIS_OS_NAME == 'osx' ]]; then export PATH=$PATH:$HOME/Library/Python/2.7/bin; fi
+  - make lint
```

Comment:
I did this step basically so that we don't block up the build queues for ages if we have lint errors.  Because some of us (me!) don't always run flake8 before pushing.